build.sbt

```diff
@@ -13,9 +19,10 @@ resolvers += Resolver.sonatypeRepo("releases")
 resolvers += Resolver.sonatypeRepo("snapshots")
 
 libraryDependencies ++= Seq(
-  "org.virtuslab" %% "unicorn" % "1.2.0",
-  "org.scalatest" %% "scalatest" % "2.2.5" % "test",
-  "com.typesafe.play" %% "play-test" % "2.5.19" % "test",
+  "org.virtuslab" %% "unicorn" % "1.3.0-SNAPSHOT",
```

Comment:
Please change to 1.3.0 as new unicorn was already released :tada:

Reply:
Changed :)